src/extension/meta.json

```diff
@@ -1,8 +1,9 @@
 {
-  "name": "Organizational Chart",
-  "description": "Chart to visualise organisations",
+  "name": "Org chart",
+  "description": "Displays the relations of an organization or similarly structured data. Starts at the top and allows for navigation through the hierachy.",
   "icon": "network",
   "preview": "preview.png",
   "author": "QlikTech International AB",
+  "supernova": true,
   "type": "visualization"
 }
```

Comment:
Would it be possible to have only meta.json or sn-org-chart.qext ?? The files are quite similar. Maybe something for after first release though

Reply:
yea, the meta file is actually only for when building with `nebula sense`, that doesn't work right now and as it won't really help us anyway I haven't pursued that. We can mostly ignore that file for now, and likely remove it later.